test/e2e_test.go

```diff
@@ -348,6 +348,22 @@ livez check passed
 			Expect(diff).To(BeEmpty(), "Unexpected metrics")
 		}
 	})
+	It("skip scrape metrics about  nodes with label node-selector filtered in cluster", func() {
+		nodeList, err := client.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+		if err != nil {
+			panic(err)
+		}
+		for _, node := range nodeList.Items {
+			node.Labels["metrics-server-skip"] = "true"
+			_, err := client.CoreV1().Nodes().Update(context.TODO(), &node, metav1.UpdateOptions{})
```

Comment:
Please remember that tests should be independent and not leave side effect. This test marks all nodes with skip label but doesn't clean the label after.

Reply:
yeah, I have updated it.